test/controllers/csv_exports_controller_test.rb

```diff
@@ -76,16 +76,87 @@
     end
 
     describe "#new" do
-      before do
-        p = projects(:test).dup
-        p.name = "Other"
-        p.save!(validate: false)
+      describe "as html" do
+        before do
+          p = projects(:test).dup
```

Comment:
Cool, would you mind adding a comment to explain that? This is a slightly unconventional way to mock ActiveRecord objects